src/api/ExperimentIdLink.js

```diff
@@ -25,17 +25,24 @@ function buildExpHeaders(cache) {
 	return experimentAssignments.join();
 }
 
-export default () => {
+export default ({ cookieStore }) => {
 	return setContext((operation, previousContext) => {
-		// Fetch experiment header value
+		// Initialize new context
+		const newContext = { ...previousContext };
+		newContext.headers = newContext.headers ?? {};
+
+		// Fetch experiment header values
 		const experimentHeader = buildExpHeaders(previousContext?.cache);
+		const uiabValue = cookieStore.get(UIAB_COOKIE_NAME) ?? '';
```

Comment:
Keep in mind that we don't actually store the assignments in the `uiab` cookie unless the assignment is forced via URL (or it's a user with an old `uiab` cookie from before the experimentation changes a few months ago).

Reply:
That's exactly what we need this for, passing the forced assignments to the api.

Reply:
Cool, good to know!